src/metaschema/oscal_poam_metaschema.xml

```diff
@@ -57,6 +57,68 @@
       </model>
    </define-assembly>
    
+   <define-assembly name="poam-items">
+      <formal-name>POA&amp;M Items</formal-name>
+      <description>This identifies initial and residual risks, deviations, and disposition.</description>
+      <model>
+         <field ref="title"  min-occurs="1" max-occurs="1" />
+         <field ref="description"  min-occurs="1" max-occurs="1" />
+         <field ref="prop"  min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="properties" in-json="ARRAY"/>
+         </field>
+         <assembly ref="annotation" min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="annotations" in-json="ARRAY"/>
+         </assembly>
+         <field ref="start" min-occurs="1" max-occurs="1" >
+            <description>Date/time stamp identifying the start of the evidence collection reflected in these results.</description>
+         </field>
+         <field ref="end" min-occurs="1" max-occurs="1">
+            <description>Date/time stamp identifying the end of the evidence collection reflected in these results. In a continuous motoring scenario, this may contain the same value as start if appropriate.</description>
+         </field>
+         <assembly ref="poam-item" min-occurs="1" max-occurs="unbounded">
+            <group-as name="poam-item-group" in-json="ARRAY"/>
+         </assembly>
+         <field ref="remarks" in-xml="WITH_WRAPPER" min-occurs="0" max-occurs="1" />
+      </model>
+   </define-assembly>
    
+   <define-assembly name="poam-item">
+      <formal-name>POA&amp;M Item</formal-name>
+      <description>Describes an individual POA&amp;M item.</description>
+      <flag ref="uuid" required="yes" />
+      <model>
+         <field ref="title"  min-occurs="1" max-occurs="1" />
+         <field ref="description"  min-occurs="1" max-occurs="1" />
+         <field ref="prop"  min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="properties" in-json="ARRAY"/>
+         </field>
+         <assembly ref="annotation" min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="annotations" in-json="ARRAY"/>
+         </assembly>
+         <field ref="date-time-stamp" min-occurs="1" max-occurs="1" >
+            <description>Date/time stamp identifying when the finding information was collected.</description>
```

Comment:
Is this (conceptually): last update to this item/finding?

Reply:
@brad, this is the date/time of the _initial_ discovery, which is critical in environments where resolution is required within a certain time frame. There is a separate ```remediation-tracking``` assembly used to log/track activity associated with resolving the finding. In theory, an entry could be added here any time data is changed in the finding. I'm open to discussing some kind of ```last-updated``` date/time field if you believe one is appropriate.

Reply:
OK, then it looks like there are (at least) two things we need to track - when the problem was found, and when the status was last updated. Possibly a "fix-by" date might also be relevant. I think the field name should describe which of those we need - the `date-time-stamp` is the syntax, but I'd prefer to see some semantic meaning encapsulated in that.